Civi/Api4/Action/Entity/Get.php

```diff
@@ -57,12 +59,14 @@ protected function getRecords() {
         foreach (glob("$dir/*.php") as $file) {
           $matches = [];
           preg_match('/(\w*)\.php$/', $file, $matches);
-          $entity = '\Civi\Api4\\' . $matches[1];
+          // Because "Case" is a reserved php keyword
+          $entityName = $matches[1] === 'CiviCase' ? 'Case' : $matches[1];
+          $className = '\Civi\Api4\\' . $matches[1];
```

Comment:
Any particular reason why this one still uses direct string manipulation?

Reply:
This is a very low-level function, and it's doing the reverse of what the new utility function does. But hang on I might have a more elegant solution...

Reply:
Ok, pushed up a better version without the specific reference to CiviCase.